### PR TITLE
beeftext: Add version 13.0

### LINKED ISSUE #7754: [Request] add beeftext
### Information

**Name:** Beeftext

**Description:** An open-source text substitution tool for Windows.

**Homepage:** https://beeftext.org

**Download link(s):** https://github.com/xmichelo/Beeftext/releases/download/v13.0/Beeftext-13.0-PortableEdition.zip

**Some indication of popularity/repute:** GitHub 474 stars

### PR DESCRIPTION
**Pull request by rkobayashi**
Closes #7754

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by HUMORCE** [comment]
re-order of fields

File: bucket/beeftext.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+    "version": "13.0",
+    "description": "An open-source text substitution tool for Windows.",
```
Comment (rashil2000) [inline bucket/beeftext.json:3]:
```suggestion
    "description": "An open-source text substitution/snippet management tool.",
```

File: bucket/beeftext.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "homepage": "https://beeftext.org/",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/xmichelo/Beeftext/releases/download/v13.0/Beeftext-13.0-PortableEdition.zip",
+            "hash": "c7357d63841485b308c096ba9732ead58880cdede996e52e149d4117d2795c85"
+        }
+    },
+    "extract_dir": "BeeftextPortableEdition",
+    "bin": "Beeftext.exe",
```
Comment (rashil2000) [inline bucket/beeftext.json:13]: Commandline arguments?

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Pull request merged**